torch/csrc/jit/passes/graph_rewrite_helper.h

```diff
@@ -1,11 +1,32 @@
 #pragma once
 
 #include <torch/csrc/jit/ir/ir.h>
+#include <torch/csrc/jit/ir/irparser.h>
 
 namespace torch {
 namespace jit {
 namespace graph_rewrite_helper {
 
+struct ConvBNParameters {
+  at::Tensor conv_w;
+  at::Tensor conv_b;
+  at::Tensor bn_rm;
+  at::Tensor bn_rv;
+  double bn_eps = 0.0;
+  at::Tensor bn_w;
```

Comment:
could you copy these to the new file instead of moving these to helper? since this will not be used by other passes after the workaround is removed

Reply:
I cannot quite move it because it is used by BatchNormConv2dFolderHelper in quantization.cpp. I am just reusing bunch of stuff you have in quantization.cpp.

Reply:
I mean could you copy the code to the new file?

Reply:
You mean the batch norm folding in quantization.cpp? Sure, but let me do it in a different diff.